### ISSUE TITLE
usb memory is not accessible , after Hello world !

### ISSUE THREAD

**Issue by faridtm** [brandonlw/Psychson#22]
i run hello world, but i have question !
after running the payload the thump drive memory appearing in windows explorer, but i can't access the usb memory space ! is it my fault, or in normal condition it is not accessible after payload injection in firmware ?

**Comment by cubic3d** [comment]
See https://github.com/adamcaudill/Psychson/blob/master/firmware/main.c

The custom firmware does not implement any flash memory access and controls just the PS2303. The memory chips are not identical in different PS2303 devices. It would be necessary to create a custom firmware for every single device.

**Comment by faridtm** [comment]
Thanks for you help, and by any chance, don't you have a firmware that do this for patriot super sonic 8GB, or is there any document, about how to make a custom firmware that implement access to flash memory and act as keyboard ?
because i don't have any idea about how to change the main.c for this purpose !

**Comment by cubic3d** [comment]
Unfortunately not. This isn't that easy, you really have to implement the whole communication part with the memory unit and processing data.

**Comment by Bonjour123** [comment]
Yes, actually, if you replace FWdummy by you USB key firmware, and then built bat and the embed payload, we should have access to the memory while having the HID payload. But I don't know why it doesn't work. 
BUT IT CAN WORK, look at the Adam Caudill conference, you will see that he has the HID fonctionnality AND access to the memory. 
If somebody has any info about this, pleuse share :d

**Comment by Bonjour123** [comment]
Sorry, I mistook, actually, he can access to the memory for the two others patch but not for the hid payload
